messaging.rst

```diff
@@ -25,25 +25,25 @@ Invariants
-|  channel_identifier   | uint256     | Channel identifier inside the TokenNetwork contract        |
-+-----------------------+-------------+------------------------------------------------------------+
-| token_network_address | address     | Address of the TokenNetwork contract                       |
-+-----------------------+-------------+------------------------------------------------------------+
-| chain_id              | uint256     | Chain identifier as defined in EIP155                      |
-+-----------------------+-------------+------------------------------------------------------------+
-|  additional_hash      | bytes32     | Computed from the message. Used for message authentication |
-+-----------------------+-------------+------------------------------------------------------------+
-|  signature            | bytes       | An elliptic curve 256k1 signature                          |
-+-----------------------+-------------+------------------------------------------------------------+
++-----------------------+-------------+-------------------------------------------------------------------------------+
+| Field Name            | Field Type  |  Description                                                                  |
++=======================+=============+===============================================================================+
+|  nonce                | uint64      | Strictly monotonic nonce                                                      |
++-----------------------+-------------+-------------------------------------------------------------------------------+
+|  transferred_amount   | uint256     | counter of tokens sent                                                        |
++-----------------------+-------------+-------------------------------------------------------------------------------+
+|  locksroot            | bytes32     | Root of merkle tree of all pending lock lockhashes                            |
++-----------------------+-------------+-------------------------------------------------------------------------------+
+|  channel_identifier   | uint256     | Channel identifier inside the TokenNetwork contract                           |
++-----------------------+-------------+-------------------------------------------------------------------------------+
+| token_network_address | address     | Address of the TokenNetwork contract                                          |
++-----------------------+-------------+-------------------------------------------------------------------------------+
+| chain_id              | uint256     | Chain identifier as defined in EIP155                                         |
++-----------------------+-------------+-------------------------------------------------------------------------------+
+|  additional_hash      | bytes32     | Hash of additional data used on the application layer. e.g.: payment metadata |
```

Comment:
Comma before *e.g.*

Reply:
Comman instead of the full stop you mean?

Reply:
yes